internal/ocm/internalid.go

```diff
@@ -67,6 +68,12 @@ func (id *InternalID) validate() error {
 		return nil
 	}
 
+	if match, _ = path.Match(aroHcpV1Alpha1NodePoolPattern, id.path); match {
+		// Temporarily use cmv1 constant for backward-compatibility.
```

Comment:
can you explain this comment a bit more - what would be our path forward for this, and how long is "temporary"?

Reply:
It's a bit of a mess still from the partial transition to the `aro_hcp` API.  **Cluster credentials** is the last thing still using the `clusters_mgmt` API.  As soon as CS makes that available through `aro_hcp` (maybe they have already... need to check) I'll come back and remove all trace of `clusters_mgmt`, including these "kind" constants.

Reply:
@mbarnes not yet afaik.